### PR TITLE
[OSDOCS-9639]Issue in file osd_architecture/osd_policy/osd-service-definition.adoc

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.14+
Issue:

https://issues.redhat.com/browse/OSDOCS-9639
Link to docs preview:

https://71502--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_architecture/osd_policy/osd-service-definition#instance-types_osd-service-definition
QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by mletalie** [comment]
Hello @ckandag,
    When you get a moment can you please provide a review for this PR? Thank you.

File: modules/sdpolicy-am-compute.adoc [hunk 1]
```diff
@@ -9,7 +9,10 @@ Single availability zone clusters require a minimum of 2 worker nodes for Custom
 
 Multiple availability zone clusters require a minimum of 3 worker nodes for Customer Cloud Subscription (CCS) clusters, 1 deployed to each of 3 availability zones. A minimum of 9 worker nodes are required for standard clusters. These 9 worker nodes are included in the base subscription, and additional nodes must be purchased in multiples of 3 to maintain proper node distribution.
 
-Worker nodes must all be the same type and size within a single {product-title} cluster.
+[NOTE]
+====
+Worker nodes must all be the same type and size within a single  {product-title} machine pool, but can be different types and sizes across multiple machine pools within an {product-title} cluster.
```
Comment (mramendi) [inline modules/sdpolicy-am-compute.adoc:14]: I would suggest this reads a bit awkwardly. Possible suggestions:

All worker nodes within a single {product-title} machine pool must be of the same type and size. However, worker nodes across multiple machine pools within an {product-title} cluster can be of different types and sizes.

If the reordering does not work I would at least add the "of" in two places.

**Review by mramendi — COMMENTED** [review]
A style nit, not critical. As I am in peer review induction @JoeAldinger will finish the review.

File: modules/sdpolicy-am-compute.adoc (hunk 1 above)
Comment (mletalie) [reply modules/sdpolicy-am-compute.adoc:14]: Thanks @mramendi, I was not really happy with what I had but was drawing a blank, so I appreciate it!

**Review by JoeAldinger — APPROVED** [review]
/lgtm

**Review by ckandag — APPROVED** [review]

**Pull request merged**